docs/docsite/rst/network/getting_started/first_inventory.rst

```diff
@@ -206,17 +206,20 @@ Copy this output into your inventory file under ``[vyos:vars]``, which now looks
 
 .. code-block:: yaml
 
-   [vyos:vars]
-   ansible_connection=network_cli
-   ansible_network_os=vyos
-   ansible_user=my_vyos_user
-   ansible_ssh_pass= !vault |
-          $ANSIBLE_VAULT;1.2;AES256;my_user
-          66386134653765386232383236303063623663343437643766386435663632343266393064373933
-          3661666132363339303639353538316662616638356631650a316338316663666439383138353032
-          63393934343937373637306162366265383461316334383132626462656463363630613832313562
-          3837646266663835640a313164343535316666653031353763613037656362613535633538386539
-          65656439626166666363323435613131643066353762333232326232323565376635
+  ...
+
```

Comment:
The line above this says:
Copy this output into your inventory file under [vyos:vars], which now looks like this:

But the updated example doesn't seem to use [vyos:vars] anymore?  Should that sentence be updated as well to show the new structure?

Reply:
changed the text and added more about INI and inlined vaults, also corrected some code block formatting